packages/terra-action-header/tests/wdio/action-header-spec.js

```diff
@@ -114,16 +79,11 @@ describe('ActionHeader', () => {
     Terra.should.matchScreenshot({ viewports });
     Terra.should.themeEachCustomProperty(
       {
-        '--terra-action-header-background-color': '#551A8B',
-        '--terra-action-header-border-bottom': '1px solid #00FF00',
-        '--terra-action-header-border-left': '1px solid #00FF00',
-        '--terra-action-header-border-right': '1px solid #00FF00',
-        '--terra-action-header-border-top': '1px solid #00FF00',
-        '--terra-action-header-box-shadow': '0 1px 0 0 #FF8C00',
-        '--terra-action-header-font-color': '#4682B4',
-        '--terra-action-header-font-family': 'Roboto',
-        '--terra-action-header-font-size': '0.5rem',
-        '--terra-action-header-padding': '1rem',
+        '--terra-action-header-margin-left': '2rem',
```

Comment:
These should be removed. They were tested above.

Reply:
Removed [here](https://github.com/cerner/terra-core/commit/98ed32090a91a4cc4aeb54e121d94d0aa060c2c4).